### ISSUE TITLE
List<Book> in body has extra "schema" causing swagger error

### ISSUE THREAD

**Issue by colbroth** [abelsilva/swaggerwcf#117]
In version 0.2.4, List<Book> in body has extra "schema" causing swagger error

I added the following to the IStore.cs file

```
        [OperationContract]
        [WebInvoke(Method = "POST")]
        void BookTest(List<Book> books);
```

The swagger.json file has the following:

```
"/v1/rest/BookTest": {
      "post": {
        "operationId": "SwaggerWcf.Test.Service.BookStore.BookTest",
        "consumes": [
          "application/json",
          "application/xml"
        ],
        "produces": [
          "application/json",
          "application/xml"
        ],
        "parameters": [
          {
            "name": "books",
            "in": "body",
            "required": true,
            "schema": {
              "type": "array",
              "items": {
                "schema": {
                  "$ref": "#/definitions/SwaggerWcf.Test.Service.Data.Book"
                }
              }
            }
          }
        ],
        "responses": {
          "default": {
            "description": "not available"
          }
        }
      }
    },
```

Expected:

```
"/v1/rest/BookTest": {
      "post": {
        "operationId": "SwaggerWcf.Test.Service.BookStore.BookTest",
        "consumes": [
          "application/json",
          "application/xml"
        ],
        "produces": [
          "application/json",
          "application/xml"
        ],
        "parameters": [
          {
            "name": "books",
            "in": "body",
            "required": true,
            "schema": {
              "type": "array",
              "items": {
                "$ref": "#/definitions/SwaggerWcf.Test.Service.Data.Book"
              }
            }
          }
        ],
        "responses": {
          "default": {
            "description": "not available"
          }
        }
      }
    },
```

Notice the additional "schema" after "items" was removed.

**Comment by abelsilva** [comment]
kinda tried to fix without testing, can you check with release `0.2.5`?

**Comment by colbroth** [comment]
The extra schema was removed but the $ref is incorrect. 

With the update, I get 

$ref: '#/definitions/'

expected 
"$ref": "#/definitions/SwaggerWcf.Test.Service.Data.Book"

**Comment by abelsilva** [comment]
pushed new version to nuget: `0.2.7`

**Comment by colbroth** [comment]
Fixed in version 0.2.7. Thank you
